packages/admin/README.md

```diff
@@ -63,10 +63,18 @@ The SDK will also provide TS typings out of the box. Please, refer to the implem
 
 ## Developer commands
 
+### `derive-airnode-xpub`
```

Comment:
@bbenligiray I've added this as it seems useful. Agree?

Reply:
💯 This seems really useful to me. I think we should add at least 1 e2e test for this new command